config/galaxy.ini.sample

```diff
@@ -1031,6 +1031,11 @@ use_interactive = True
 # if running many handlers.
 #cache_user_job_count = False
 
+#Extra directories to be mounted inside Docker containers. Note that ALL docker containers
+#will mount these volumes
+#extra_docker_volumes = /opt/Genomics:ro,/cluster_share/Galaxy/database/results:rw
```

Comment:
Correct - you should add the configuration variable in your universe_wsgi.ini file (not in the config/galaxy.ini.sample file - just an example here).
Note that you do NOT need to include the following as they will be included by default:
database/job_working_directory
database/files
galaxy   [galaxy directory]
This configuration variable is only for non-standard directories that Galaxy knows nothing about - /opt/Genomics, for example.